lib/Processor.ts

```diff
@@ -47,8 +53,19 @@ export default class Processor extends EventEmitter {
   }
 
   private async handleConsumerMessage(message: ConsumerPayloadInterface): Promise<void> {
+    const logPayload = {identifier: message.key.toString(), url: message.url};
+
+    this.logger.info("starting crawling", logPayload);
+
+    const startTime = new Date().getTime();
```

Comment:
best practice for measuring execution-time is to use [process.hrtime](https://nodejs.org/api/process.html#process_process_hrtime). But feel free to leave this as-is, as I'd suggest to later add metrics via prometheus nevertheless.